packages/cli/tslint.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "tslint-config-airbnb",
+  "rules": {
+    "file-header": [
+      true,
+      {
+        "match": "Copyright \\d{4}",
+        "allow-single-line-comments": true,
+        "default": "Copyright 2019",
+        "enforce-trailing-newline": false
```

Comment:
je penses qu'il faut le forcer à true pour cette raison : https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file